components/base/Collapsible.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsx jsx */
+import ReactCollapsible from "react-collapsible";
+import styled from "@emotion/styled";
+import { css, jsx } from "@emotion/core";
+import plusIcon from "../../public/images/plus-circle-icon.svg";
+import minusIcon from "../../public/images/minus-circle-icon.svg";
+import { GREY_2, WHITE_PRIMARY } from "../colors";
+import { POPPINS } from "../typography";
+import { BOX_SHADOW_7 } from "../shadowDepths";
+
+// Styles
+const CollapsibleWrapper = styled.div`
+  /* Main container element for react-collapsible https://www.npmjs.com/package/react-collapsible#collapsible */
+  .Collapsible {
+    padding: 26px 30px;
+    box-sizing: border-box;
+    width: 100%;
+    border: 1px solid ${GREY_2};
+    border-radius: 5px;
+    background-color: ${WHITE_PRIMARY};
+    font-family: ${POPPINS};
+    font-size: 18px;
+    line-height: 27px;
+    cursor: pointer;
```

Comment:
I noticed that the cursor changes if you hover anywhere on the component, but only toggles if you click on the 'question' or the icon. Can we make it so that either the cursor only changes when hovering over that part of the component or clicking anywhere on the card triggers the toggle?

I think being able to click anywhere would be a better solution and more consistent with our other card-like components, but that might cause problems if the answer has a link in it (?), so I'd be fine with either!

Reply:
I like the the latter too: being able to click anywhere in the card. I remember trying other ways to make the whole card clickable but failed. Perhaps the padding of the container is what's causing it to not be clickable. 🤔  I'll tinker around with that, and also try it with adding links to the answers as well! Thanks for the review, Connor!

Reply:
This [commit](https://github.com/Dwolla/dev-portal/pull/41/commits/d0d3001d409ff380c5db788e1f385048dcd05363) should take care of that! 
Instead of overriding  the padding style of the container, I ended up adding that padding to the trigger prop that's getting passed to the package, which seems to be working to let us click anywhere on the closed card!

Reply:
Ahhh, nice fix for that! One last small thing, can you move `cursor: pointer` from `.Collapsible` to the `StyledTrigger` styles? It still looks like clicking on the bottom part of the open card will do something when it doesn't

Reply:
Ah, good catch! Missed that! 😅 This [commit](https://github.com/Dwolla/dev-portal/pull/41/commits/86b56e7f9ddd7d8efe5e265693bd20e5c0accac3) should fix that! :)